Formula/verapdf-apps.rb

```diff
@@ -0,0 +1,44 @@
+class VerapdfApps < Formula
+  desc "Open-source industry-supported PDF/A validation"
+  homepage "https://verapdf.org/home/"
+  url "https://github.com/veraPDF/veraPDF-apps/archive/refs/tags/v1.21.4.tar.gz"
+  sha256 "b926c397f98c03c506e486593e0224f4250c5329ed353a280e4dcfa26ca09b68"
+  license any_of: ["GPL-3.0-or-later", "MPL-2.0"]
+  head "https://github.com/veraPDF/veraPDF-apps.git", branch: "integration"
+
+  depends_on "maven" => :build
+  depends_on "openjdk" => :build
+
+  def install
+    system "mvn", "clean", "install"
+
+    installer_file = Pathname.glob("installer/target/verapdf-izpack-installer-*.jar").first
+    auto_install_path = Pathname("installer/target/auto-install.xml")
+    auto_install_path.write <<~EOS
+      <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+      <AutomatedInstallation langpack="eng">
+          <com.izforge.izpack.panels.htmlhello.HTMLHelloPanel id="welcome"/>
+          <com.izforge.izpack.panels.target.TargetPanel id="install_dir">
+              <installpath>#{prefix}</installpath>
+          </com.izforge.izpack.panels.target.TargetPanel>
+          <com.izforge.izpack.panels.packs.PacksPanel id="sdk_pack_select">
+              <pack index="0" name="veraPDF GUI" selected="true"/>
+              <pack index="1" name="veraPDF Mac and *nix Scripts" selected="true"/>
+              <pack index="2" name="veraPDF Corpus and Validation model" selected="false"/>
+              <pack index="3" name="veraPDF Documentation" selected="true"/>
+              <pack index="4" name="veraPDF Sample Plugins" selected="false"/>
+          </com.izforge.izpack.panels.packs.PacksPanel>
+          <com.izforge.izpack.panels.install.InstallPanel id="install"/>
+          <com.izforge.izpack.panels.finish.FinishPanel id="finish"/>
+      </AutomatedInstallation>
+    EOS
```

Comment:
This looks rather difficult to maintain. Is there not something upstream we can use? Or can this be upstreamed instead?

Reply:
There's an easier way to change the install prefix, but there's no easier way to change the pack options — I already discussed this with the developers.